### ISSUE TITLE
Switch GitHub PR test from ant precommit to gradle (Jira: LUCENE-9146)

### ISSUE THREAD

**Issue by mocobeta** [mocobeta/sandbox-lucene-10557#226]
---
### Jira information

Original Jira: https://issues.apache.org/jira/browse/LUCENE-9146
Reporter: Mike Drob
Assignee: Anshum Gupta
Created: 2020-01-16T21:40:34.149+0000
Updated: 2020-02-20T11:29:27.082+0000
Resolved: 2020-02-20T11:29:27.006+0000

Issue Links:

Sub-Tasks:

Pull Requests:

**Comment by mocobeta** [comment]
I think it can run both, at least at the beginning?

Author: Dawid Weiss (`@dweiss`)
Created: 2020-01-26T17:13:48.560+0000
Updated: 2020-01-26T17:13:48.560+0000

**Comment by mocobeta** [comment]
Yes. I think it makes sense to leave the ant variant where it is and add the gradle version for a couple of minor releases, or until 9 (whatever is sooner). It doesn't cost anything and would just run in parallel.

Author: Anshum Gupta
Created: 2020-02-04T09:28:39.030+0000
Updated: 2020-02-04T09:28:39.030+0000

**Comment by mocobeta** [comment]
Commit 7c20f6b8c5ec46cdd3f8f32a2fedcb5b0406ba3b in lucene-solr's branch refs/heads/master from Anshum Gupta
[ https://gitbox.apache.org/repos/asf?p=lucene-solr.git;h=7c20f6b ]

LUCENE-9146 (#226): Create gradle precommit action (#1245)



Author: ASF subversion and git services
Created: 2020-02-07T23:10:25.600+0000
Updated: 2020-02-07T23:10:25.600+0000

**Comment by mocobeta** [comment]
Merged into master.

Author: Anshum Gupta
Created: 2020-02-07T23:10:55.736+0000
Updated: 2020-02-07T23:10:55.736+0000

**Comment by mocobeta** [comment]
I noticed chmod +x on gradlew - why is that needed? It should be set by git (as it's checked in with proper executable bit).

Author: Dawid Weiss (`@dweiss`)
Created: 2020-02-13T08:43:32.702+0000
Updated: 2020-02-13T08:43:32.702+0000

**Comment by mocobeta** [comment]
Can we close this?

Author: Jan Høydahl (`@janhoy`)
Created: 2020-02-13T13:12:18.901+0000
Updated: 2020-02-13T13:12:18.901+0000

**Comment by mocobeta** [comment]
I'm not sure why this is in the workflow:
```Java

   - name: Grant execute permission for gradlew
      run: chmod +x gradlew 
```

Author: Dawid Weiss (`@dweiss`)
Created: 2020-02-13T13:20:24.773+0000
Updated: 2020-02-13T13:20:24.773+0000
